project2/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+    <title>Movie Card</title>
+</head>
+
+<body>
+    <div class="card">
+        <div class="transbox">
+            <h2>THE GRAND BUDAPEST HOTEL</h2>
```

Comment:
You want to avoid using ALL CAPS in your HTML file and instead apply `text-transform: uppercase;` in your CSS:
```suggestion
            <h2>The Grand Budapest Hotel</h2>
```

More info on [`text-transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform).